wa/instruments/perf.py

```diff
@@ -77,6 +77,17 @@ class PerfInstrument(Instrument):
                   description="""Provides labels for pref output. If specified, the number of
                   labels must match the number of ``optionstring``\ s.
                   """),
+        Parameter('perf_mode', kind=str, default='stat', allowed_values=['stat', 'record'],
+                  global_alias='perf_mode',
+                  description="""
+                  Choose between 'perf stat' and 'perf record'. If 'perf record' is selected
+                  'perf report' will also be run. 'report_optionstring' can be used to generate
+                  custom report from the trace.
+                  """),
+        Parameter('report_optionstring', kind=list_or_string, default='',
+                  global_alias='report_perf_options',
+                  description="""Specifies options to be used for the 'perf report' command used
```

Comment:
Please format consistently with the other parameters.

Reply:
Thanks, changed in latest version.